karen-tracy/scripts/articleView.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+let articleView = {};
+
+// DONE: Where possible, refactor methods into arrow functions, including the document.ready() method at the bottom.
+
+// COMMENT: How do arrow functions affect the context of "this"? How did you determine if a function could be refactored?
+// It changes the scope  to one parent above, so changes the context of this to the parent scope. If there's a contextual this then the arrow function will change the scope and will not work.
+
+articleView.populateFilters = () => {
+
+  let optionTemplateScript = $('#option-template').html();
+  let optionTemplate = Handlebars.compile(optionTemplateScript);
+  let val;
```

Comment:
You don't need to define this variable outside of the function, since you only use it within the function; you could just change line 19 to `let val = $(this).data('author');` and remove this line entirely.